src/VirtualClient/VirtualClient.Contracts/Enumerations.cs

```diff
@@ -677,6 +677,27 @@ public enum MetricRelativity
         LowerIsBetter = 2
     }
 
+    /// <summary>
+    /// Metric verbosity to mark the importance of a metric.
+    /// </summary>
+    public enum MetricVerbosity
```

Comment:
I would prefer to use the existing LogLevel enumeration so that we don't need to reinvent the wheel here. It has all of the terms we would need.

e.g.
Verbosity=Trace
Verbosity=0

Verbosity=Information
Verbosity=2

Additionally, the string representation as well as the numeric representation of the enum should work.

Reply:
Switched to loglevel